rxandroidble/src/main/java/com/polidea/rxandroidble2/exceptions/BleCharacteristicNotFoundException.java

```diff
@@ -9,14 +9,30 @@
  */
 public class BleCharacteristicNotFoundException extends BleException {
 
-    private final UUID charactersisticUUID;
+    private final UUID characteristicUUID;
 
-    public BleCharacteristicNotFoundException(UUID charactersisticUUID) {
-        super("Characteristic not found with UUID " + charactersisticUUID);
-        this.charactersisticUUID = charactersisticUUID;
+    public BleCharacteristicNotFoundException(UUID characteristicUUID) {
+        super("Characteristic not found with UUID " + characteristicUUID);
+        this.characteristicUUID = characteristicUUID;
     }
 
+    /**
+     * Returns characteristic UUID that has not been found
+     * @deprecated Use {@link #getCharacteristicUUID()}
+     *
+     * @return the UUID
+     */
+    @Deprecated
     public UUID getCharactersisticUUID() {
```

Comment:
missed one

Reply:
It is now deprecated. I cannot delete it until `2.0.0` as it would be a breaking change. It is advised to use getter without the typo.